src/openms/source/FILTERING/DATAREDUCTION/IsotopeDistributionCache.cpp

```diff
@@ -40,89 +40,116 @@
 namespace OpenMS
 {
 
-  IsotopeDistributionCache::IsotopeDistributionCache(double max_mass, double mass_window_width, double intensity_percentage, double intensity_percentage_optional) :
-    mass_window_width_(mass_window_width)
+  IsotopeDistributionCache::IsotopeDistributionCache()
   {
-    Size num_isotopes = std::ceil(max_mass / mass_window_width) + 1;
+    mass_window_width_ = 20.0;
```

Comment:
move initialization to initializer list (as done for `mass_window_width_(mass_window_width)` before)